src-arduino/examples/read_rfid/read_rfid.ino

```diff
@@ -13,7 +15,9 @@ void setup()
   while (!Serial)
     ; // Do nothing if no serial port is opened (added for Arduinos based on ATMEGA32U4)
   Serial.println("Testing out connecting to SPI ");
-  SPI.begin(); // Init SPI bus
+  
+  SPI.begin(7, 11, 9, 5); // Init SPI bus
+  // SPI.begin(); // Init SPI bus
```

Comment:
Any reason for this comment to be here?

Reply:
the default begin uses the default pins, leaving just as a way to get back